updating/updating-restricted-network-cluster/index.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="about-restricted-network-updates"]
+= About disconnected environment updates
+include::_attributes/common-attributes.adoc[]
+:context: about-restricted-network-updates
+
+toc::[]
+
+A restricted network environment is one in which your cluster nodes cannot access the internet. 
```

Comment:
We are using both `disconnected` and `restricted network`. I'm trying to understand the difference between them. So when we use `disconnected`, we are talking about the cluster. While we use `restricted network`, we are talking about the network. Am I right?

Reply:
So disconnected environment is the preferred term per #48887, and I changed the headings to that use term in an attempt to accommodate SME feedback while saving content updates for a single PR later down the line.

However, I'll just make the changes now so that every instance of `restricted network` is updated to say `disconnected environment` instead. Thanks!